app/views/contexts/hub_context.py

```diff
@@ -151,7 +163,9 @@ def _get_row_for_section(
             row_id,
         )
 
-    def _get_rows(self, enabled_section_ids) -> List[Mapping[str, Union[str, List]]]:
+    def _get_rows(
+        self, enabled_section_ids: List[str]
```

Comment:
This one is still outstanding, "For standard collections, use lower case names e.g `list` rather than `List`". There are other places where it needs fixing.

Reply:
👍